### DISCUSSION TITLE
Better cursor for Argon "pro" skin for readability

### DISCUSSION THREAD

**Discussion by NeekoVN** [ppy/osu#22061] [Ideas]
As it's a "pro" variant of the Argon skin, I really think it's a good idea to at least, change the cursor trail to something a bit easier to read because myself and I believe many others too are having a hard time trying to find the cursor's location with all those lightings and a tiny trail, especially in tech maps or those with very fast jumps.

https://user-images.githubusercontent.com/76953348/211144352-e7360688-48ea-4dc1-b2f4-7b553e69f0de.mp4

![argon cursor](https://user-images.githubusercontent.com/76953348/211144601-ef51a7c1-0075-4a62-95d5-15e5697c9b8f.png)

**Comment by peppy** [comment]
Thanks for the feedback.

**Comment by mk56-spn** [comment]
This seems purely subjective. There's pro skins with no trail ( rohulk v 4 ) . This sort of thin trail ( idke )  And skins with the " fat trail" ( most rafis variants come to mind  )  . But in any case what sort of trail did you have in mind that would fit better

**Reply by NeekoVN** [reply]
I'm thinking of a shorter trail with (almost) the same width as the cursor. Yes it's subjective but that's the type of cursor that most of the players are familiar with, I'm not asking for that exact cursor you're probably thinking about but I hope the Osu! contributors can get creative   in making a cursor both unique and familiar.

Or just have another option to customize the cursor idk 💀

**Reply by Speykious** [reply]
Customizable trail length, width and color would ultimately be a good feature to have, I think.
Kinda how Danser does it.

Currently trails are just repeated images, and in most skins that image is a blurry circle.

**Reply by mk56-spn** [reply]
> Customizable trail length, width and color would ultimately be a good feature to have, I think. Kinda how Danser does it.
> 
> Currently trails are just repeated images, and in most skins that image is a blurry circle.

this all sounds nice to have, and *might* happen at some point, but not anytime soon in my estimation, core devs are very busy, and theres already a massive backlog of contributor PRs to go through as it stands, so even third parties chipping in probably wont make this come to fruition promptly

**Reply by Speykious** [reply]
Oh it's ok I'm aware of that. Ngl osu!lazer is like the most overwhelmingly active project I've ever seen on GitHub. xD
